app/javascript/components/navbar/ExamNavbar.tsx

```diff
@@ -20,41 +22,58 @@ import Scratch from '@hourglass/containers/navbar/Scratch';
 import ExamMessages from '@hourglass/containers/navbar/ExamMessages';
 import AskQuestion from '@hourglass/components/navbar/AskQuestion';
 
-
 interface NavAccordionItemProps {
+  onSectionClick: (eventKey: string) => void;
+  expanded: boolean;
   Icon: IconType;
   label: string;
   className?: string;
   eventKey: string;
   direction?: 'up' | 'down';
 }
 
+const iconSize = '1.5em';
```

Comment:
Should we make `1.5em` a project-wide constant for icon sizes?

Reply:
Probably, yes.  Where ought that go?

Reply:
`helpers.ts` is kind of a catchall right now and is probably going to be split up later..

Maybe we want a `@hourglass/constants` with `export const ICON_SIZE = '1.5em';`

Reply:
That sounds reasonable

Reply:
I think the linter will complain about wanting you to use the default export - just add a `// eslint-disable-next-line import/prefer-default-export` before the `ICON_SIZE` line until we have more constants.